ingestion_server/test/integration_test.py

```diff
@@ -464,16 +465,17 @@ def test_task_count_after_two(self):
         self.assertEqual(2, len(res_json), msg)
 
     def _get_es(self):
-        return Elasticsearch(
-            host="localhost",
-            port=service_ports["es"],
-            connection_class=RequestsHttpConnection,
-            timeout=10,
+        endpoint = f"http://localhost:{service_ports['es']}"
+        es = Elasticsearch(
+            endpoint,
+            node_class=RequestsHttpNode,
+            request_timeout=10,
```

Comment:
Why was `timeout` changed to `request_timeout` here? These seem to be different values.

Also, the `http_auth` from line 475 is [deprecated](https://www.elastic.co/guide/en/elasticsearch/client/python-api/master/migration.html#migration-options), and should be replaced with `basic_auth` (I don't know if it has any effect when the value is None, though)

Reply:
`timeout` does not exist any more. `request_timeout` configures the length of time to wait for a request to ES before timing out: https://www.elastic.co/guide/en/elasticsearch/client/python-api/master/config.html#timeouts

Here's the documentation for the old `timeout` parameter, which appears to do the same thing, but by a different name: https://elasticsearch-py.readthedocs.io/en/v7.17.11/api.html?highlight=timeout#timeout

Both configure the global request timeout for all requests, as far as I can tell? Are you finding anything different that I'm missing?

Reply:
Elasticsearch does not support AWS auth directly, and trying to pass it to `basic_auth` breaks the integration. I've found a nasty workaround, but it would probably be much better to find out if we can remove this bit altogether.

Reply:
Removed in https://github.com/WordPress/openverse/pull/3029/commits/f138fec3e865d5e1f4959c436c8f435a56259286

Reply:
Sorry, you're right.  I was reading the `__init__` code of the Elasticsearch class, and saw both `timeout` and `request_timeout` there, but didn't see the check that shows that `timeout` is deprecated below that: `"The 'timeout' parameter is deprecated in favor of 'request_timeout'"`.
Both seem to configure the global request timeouts for all requests at `init` time. It is also possible to set a different `request_timeout` for the specific request using `options`: `es.options(request_timeout=5).search()`: https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/config.html#timeouts

> I'll go ahead and remove it and we can see whether staging is still able to communicate with Elasticsearch (I really can't see why it wouldn't, but I could very easily be missing something) and roll back to the workaround if staging fails.
So, the `auth` failure would be something to watch for when deploying this change, right?